.github/workflows/php.yml

```diff
@@ -27,4 +28,4 @@ jobs:
         run: composer install --no-progress
 
       - name: Run PHPUnit
-        run: vendor/bin/phpunit
+        run: vendor/bin/phpunit --coverage-text
```

Comment:
See comment about coverage above. Currently the result is just printed in the action run.